### PR TITLE
#2438 - feat(analysisrun): create model and migration

### PR DESCRIPTION
**Pull request by adrian-lara**
#### Any background context you want to provide?
Part of the larger Analysis epic

#### What's this PR do?
Create the AnalysisRun model

#### How should this be manually tested?
Run migration

#### What are the relevant tickets?
#2438

### CODE REVIEW

File: seed/models/analysis_runs.py [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+        (RUNNING, 'Running'),
+        (FAILED, 'Failed'),
+        (STOPPED, 'Stopped'),
+        (COMPLETED, 'Completed'),
+    )
+
+    analysis = models.ForeignKey(Analysis, on_delete=models.CASCADE)
+    cycle = models.ForeignKey(Cycle, on_delete=models.CASCADE)
+    property = models.ForeignKey(Property, on_delete=models.CASCADE)
+    property_state = models.ForeignKey(PropertyState, on_delete=models.CASCADE)
```
Comment (adrian-lara) [inline seed/models/analysis_runs.py:36]: When deciding to use CASCADE, I went with the approach of completing the following sentence: "If I delete the related [insert FK model], AnalysisRun should... ".

I ended up with "... also be deleted" myself, but I'm open to opposing thoughts.

File: seed/models/analysis_runs.py [hunk 2]
```diff
@@ -0,0 +1,38 @@
...
+
+class AnalysisRun(models.Model):
+
+    CREATING = 10
+    READY = 20
+    QUEUED = 30
+    RUNNING = 40
+    FAILED = 50
+    STOPPED = 60
+    COMPLETED = 70
```
Comment (adrian-lara) [inline seed/models/analysis_runs.py:21]: @macintoshpie I really like the idea of going with 10, 20, etc. future flexibility. Thanks for suggesting it!

**Review by macintoshpie — APPROVED** [review]
Just one comment, but I think this is good to go 👍

**Review by macintoshpie — APPROVED** [review]
Awesome 🐞 🐞

**Pull request merged**